spec/controllers/concerns/client_sortable_spec.rb

```diff
@@ -64,6 +80,19 @@ def filter_cookie_name
       end
     end
 
+    # always-on filters should be merged with filters in client_sortable
+    # and always-on should be set to { assigned_to_me: true } when @always_current_user_assigned == true
+    # context "always-on filters" do
+    #   before do
+    #     subject.instance_variable_set(@always_on_filters, { assigned_to_me: true })
+    #   end
+    #
+    #   it "merges the always-on filters with the filters" do
+    #     subject.filtered_and_sorted_clients
+    #     expect(assigns(:filters)[:assigned_to_me]).to be_present
+    #   end
+    # end
+
```

Comment:
@embarnard should this still be commented out? Can it be returned or deleted?

Reply:
yeah we should just remove this